src/screens/SiteDetailScreen.js

```diff
@@ -114,6 +114,74 @@ class SiteDetailScreen extends Component {
         });
     };
 
+    renderAlternateSites = () => {
+        const {selectedSite} = this.props;
+        const {sectionTitleStyle, textStyle, bottomMargin} = styles;
+
+        if (selectedSite && selectedSite.alternateSites) {
+            return (
+                <View>
+                    <Text style={sectionTitleStyle}>
+                        {campsite_form.alternate_sites}
+                    </Text>
+                    <Text style={[textStyle, bottomMargin]}>
+                        {selectedSite.alternateSites}
+                    </Text>
+                </View>
+            );
+        }
+
+    };
+
+    renderCellProvider = ({cellProvider}) => {
+        if (cellProvider) {
+            const {textStyle, bottomMargin} = styles;
+
+            return (
+                <Text style={[textStyle, bottomMargin]}>
+                    {campsite_form.cell_provider_options[cellProvider]}
+                </Text>
+            );
+        }
+    };
+
+    renderCellStrength = ({cellProvider, cellStrength}) => {
+        if (cellStrength) {
+            const {textStyle, bottomMargin, leftPad} = styles;
+            const stylesList = cellProvider ? [textStyle, bottomMargin, leftPad] : [textStyle, bottomMargin];
+
+            return (
+                <Text style={stylesList}>
+                    {campsite_form.cell_strength_options[cellStrength]}
+                </Text>
+            );
+        }
+    };
+
+    renderCellCoverageInfo = () => {
+        const {selectedSite} = this.props;
+
+        if (selectedSite) {
+            const {cellProvider, cellStrength} = selectedSite;
+            if (cellProvider || cellStrength) {
+                const {sectionTitleStyle, cellServiceContainerStyle} = styles;
+
+                return (
+                    <View>
+                        <Text style={sectionTitleStyle}>
+                            {campsite_form.cell_service}
+                        </Text>
+                        <View style={cellServiceContainerStyle}>
+                            {this.renderCellProvider({cellProvider})}
+                            {this.renderCellStrength({cellProvider, cellStrength})}
+                        </View>
+                    </View>
```

Comment:
If the site has either `cellProvider` or `cellStrength` information, go ahead and render the whole section.